README.md

```diff
@@ -4,6 +4,22 @@
 
 To run the example project, clone the repo, and run `pod install` from the Example directory first.
 
+### Adding A New Subspec
+1. Create a new directory within the Classes folder (or Assets, if required)
+2. Add the new files to the directory created in step 1
+3. Add a subspec to the Swiftilities.podspec following this pattern:
+    ```ruby
+    # <Your Subsepc Name>
+    
+    s.subspec "<Your Subsepc Name>" do |ss|
+    	ss.source_files = "Pod/Classes/<Your Subsepc Name>/*.swift"
+    	ss.frameworks   = ["<Any Required Modules>"]
+    end 
+    ```
+4. Apend a `ss.dependency` to `s.subsec` within the podspec file with the following format: 
+`ss.dependency 'Swiftilities/<Your Subsepc Name>'`
```

Comment:
#nth: this doesn't wrap to a new line, because it's only separated by one line. Might as well put it in a fenced code block like the previous example.